admin/notifiers/class-recalibration-beta.php

```diff
@@ -7,6 +7,8 @@
 
 /**
  * Represents the logic for showing recalibration beta notice.
+ *
+ * @deprecated 9.6
```

Comment:
This won't do anything. I suggest to create a constructor with a call to `_deprecated_constructor()`

Reply:
And because we deprecate this class you can move it to the `deprecated` directory.